sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UcfgGenerator.cs

```diff
@@ -95,13 +95,16 @@ protected override void Initialize(SonarAnalysisContext context)
 
                     cc.RegisterSyntaxNodeActionInNonGenerated(
                         c => WriteUCFG<BaseMethodDeclarationSyntax>(c, x => x.Body),
-                        SyntaxKind.ConstructorDeclaration,
-                        SyntaxKind.OperatorDeclaration);
+                        SyntaxKind.ConstructorDeclaration);
 
                     cc.RegisterSyntaxNodeActionInNonGenerated(
                         c => WriteUCFG<MethodDeclarationSyntax>(c, x => (CSharpSyntaxNode)x.Body ?? x.ExpressionBody?.Expression),
                         SyntaxKind.MethodDeclaration);
 
+                    cc.RegisterSyntaxNodeActionInNonGenerated(
+                        c => WriteUCFG<OperatorDeclarationSyntax>(c, x => (CSharpSyntaxNode)x.Body ?? x.ExpressionBody?.Expression),
```

Comment:
You were already handling operators but not when using the expression body so I have updated the generation.